src/der.rs

```diff
@@ -221,7 +223,8 @@ mod tests {
           &[0x02, 0x00, 0x01],
           &[0x02, 0x01],
           &[0x02, 0x01, 0x00, 0x01],
-          &[0x02, 0x01, 0x01, 0x00], // Would be valid if last byte is ignored.
+          /* Would be valid if last byte is ignored. */
+          &[0x02, 0x01, 0x01, 0x00],
```

Comment:
Presumably this could be written with a C++ comment in that position instead of a C-style comment?